### ISSUE TITLE
Show RSS feed for the map status if available

### ISSUE THREAD

**Issue by sirKitKat** [Glodenox/wme-om#18]
When there is a rss feed with possible information about map status, provide an click-able overview below the settings.
Example: https://www.agiv.be/rss/news

Alternative, you can provide a link to a status page about the map below the settings of the selected map.
Example: https://www.agiv.be/news

**Comment by Glodenox** [comment]
Hmm, the problem there is that there is no easy way to find such news feeds for each map. I could do that manually for each map, but I intend to make it easier for people to add their own maps. I'll think about it.

**Comment by sirKitKat** [comment]
It could be an optional feed, so if no url is provided, no link shows up?

**Comment by Glodenox** [comment]
Certainly, I was just hoping it would somehow be part of the information in the GetCapabilities operation of map servers so it could be retrieved automatically, but that's not the case. I'd probably make the script retrieve the new feed items if a feed is defined as that would be easier to use.

**Comment by Glodenox** [comment]
As I'll be managing the available maps myself, this could be added in the future. If I make such functionality available, I can simply add a link to such feeds myself.
